docs/tutorial-script-single.rst

```diff
@@ -0,0 +1,280 @@
+       --address-prefixes 10.0.1.0/24 \
+       --network-security-group ha-demo-nsg
+
+Finally add four virtual machines (ha-demo-a, ha-demo-b, ha-demo-monitor, and
+ha-demo-app). For speed we background the ``az vm create`` processes and run
+them in parallel:
+
+.. code-block:: bash
+
+   # create VMs in parallel
+   for node in monitor a b app
+   do
+   az vm create \
+       --resource-group ha-demo \
+       --name ha-demo-${node} \
+       --vnet-name ha-demo-net \
+       --subnet ha-demo-subnet \
+       --nsg ha-demo-nsg \
+       --public-ip-address ha-demo-${node}-ip \
+       --image debian \
+       --admin-username ha-admin \
+       --generate-ssh-keys &
+   done
+   wait
+
+To make it easier to SSH into these VMs in future steps, let's make a shell
+function to retrieve their IP addresses:
+
+.. code-block:: bash
+
+  # run this in your local shell as well
+
+  vm_ip () {
+    az vm list-ip-addresses -g ha-demo -n ha-demo-$1 -o tsv \
+      --query '[] [] .virtualMachine.network.publicIpAddresses[0].ipAddress'
+  }
+
+Let's review what we created so far.
+
+.. code-block:: bash
+
+  az resource list --output table --query \
+    "[?resourceGroup=='ha-demo'].{ name: name, flavor: kind, resourceType: type, region: location }"
+
+This shows the following resources:
+
+::
+
+    Name                             ResourceType                                           Region
+    -------------------------------  -----------------------------------------------------  --------
+    ha-demo-a                        Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-app                      Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-b                        Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-monitor                  Microsoft.Compute/virtualMachines                      eastus
+    ha-demo-appVMNic                 Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-aVMNic                   Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-bVMNic                   Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-monitorVMNic             Microsoft.Network/networkInterfaces                    eastus
+    ha-demo-nsg                      Microsoft.Network/networkSecurityGroups                eastus
+    ha-demo-a-ip                     Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-app-ip                   Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-b-ip                     Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-monitor-ip               Microsoft.Network/publicIPAddresses                    eastus
+    ha-demo-net                      Microsoft.Network/virtualNetworks                      eastus
+
+.. _script_single_automated:
+
+Using the pg_autoctl Azure integration commands
```

Comment:
This is where an unwary person might continue copy and pasting code blocks, and try to recreate a region unnecessarily.